mod_reforged/hooks/ui/global/data_helper.nut

```diff
@@ -48,4 +48,13 @@
 		}
 		return ret;
 	}
+
```

Comment:
one newline too many. Not a big deal all things considered.
Is that on purpose or by accident?

Reply:
Looks like an accident, I'll remove it.